tests/lib/Search/Query/Common/AggregationVisitor/DispatcherAggregationVisitorTest.php

```diff
@@ -17,7 +17,7 @@
 final class DispatcherAggregationVisitorTest extends TestCase
 {
     private const EXAMPLE_LANGUAGE_FILTER = [
-        'languageCode' => 'eng-gb',
+        'languages' => ['eng-gb'],
```

Comment:
Now when you tinker with this code, maybe:
```suggestion
        'languages' => ['eng-GB'],
```

Reply:
I don't mind to change it, although as far as I can tell it doesn't really affect anything :sweat_smile:

Reply:
As you see, I approve the PR anyway. To me it just looks weird and inconsistent, but indeed does not change the result.